js/samples/04.ai.a.teamsChefBot/src/index.ts

```diff
@@ -61,7 +61,15 @@ server.listen(process.env.port || process.env.PORT || 3978, () => {
     console.log('\nTo test your bot in Teams, sideload the app manifest.json within Teams Apps.');
 });
 
-import { Application, ConversationHistory, DefaultPromptManager, DefaultTurnState, OpenAIModerator, OpenAIPlanner, AI } from '@microsoft/botbuilder-m365';
+import {
+    Application,
+    ConversationHistory,
+    DefaultPromptManager,
+    DefaultTurnState,
+    OpenAIModerator,
+    OpenAIPlanner,
+    AI
+} from '@microsoft/botbuilder-m365';
```

Comment:
This needs to be updated to `@microsoft/teams-ai`

Reply:
This wasn't the sample I authored, so I left that reference intact. But I can change it since I've already touched quite a few files with linting fixes.

Reply:
Uh I left a comment but it disappeared, so potential duplicate: 

No worries, I have this fix in a branch already and got sidetracked. Resolving :)